pabi_account_financial_report_webkit/report/common_reports.py

```diff
@@ -117,6 +117,54 @@ def _get_date_to(self, data):
     def _get_form_param(self, param, data, default=False):
         return data.get('form', {}).get(param, default)
 
+    def _get_amount_unpaid(self, lines, data):
+        Recon = self.pool.get('account.move.reconcile')
+        rec_ids = []
+        percent = []
+        account = []
+        if data['form']['filter'] == 'filter_no':
+            fiscalyear = self.get_fiscalyear_br(data)
+            specific_report = data.get('specific_report')
+            stop_period = self.get_last_fiscalyear_period(
+                fiscalyear, specific_report=specific_report)
+            date_stop = stop_period.date_stop
+        elif data['form']['filter'] == 'filter_date':
+            date_stop = self._get_form_param('date_to', data)
+        else:
+            stop_period = self.get_end_period_br(data)
+            date_stop = stop_period.date_stop
+        for line in lines:
```

Comment:
```suggestion
        rec_ids = [line['rec_id'] for line in lines if line['rec_id'] not in rec_ids]
```
วน for เพื่อต้องการ append อย่างเดียวใช้ for แบบนี้จะเร็วกว่า
และ rec_ids = [] ก็ไม่จำเป็นต้องมีด้วย